crates/swc_ecma_transforms_proposal/tests/decorators/2022-03-ordering/accessor-method-initializers/exec.js

```diff
@@ -1,69 +1,69 @@
 var log = [];
 
 function push(x) {
-  log.push(x);
-  return x;
+    log.push(x);
+    return x;
 }
 
 function logClassDecoratorRun(a, b, c) {
-  push(a);
-  return function (el, { addInitializer }) {
-    push(b);
-    addInitializer(function () {
-      push(c);
-    });
-    return el;
-  };
+    push(a);
+    return function (el, { addInitializer }) {
+        push(b);
+        addInitializer(function () {
+            push(c);
+        });
+        return el;
+    };
 }
 
 function logAccessorDecoratorRun(a, b, c, d) {
-  push(a);
-  return function (el, { addInitializer }) {
-    push(b);
-    addInitializer(function () {
-      push(c);
-    });
-    return {
-      init: () => push(d),
+    push(a);
+    return function (el, { addInitializer }) {
+        push(b);
+        addInitializer(function () {
+            push(d);
+        });
+        return {
+            init: () => push(c),
+        };
     };
-  };
 }
 
 function logMethodDecoratorRun(a, b, c, d) {
-  push(a);
-  return function (el, { addInitializer }) {
-    push(b);
-    addInitializer(function () {
-      push(c);
-    });
-    return () => (el(), push(d))
-  };
+    push(a);
+    return function (el, { addInitializer }) {
+        push(b);
+        addInitializer(function () {
+            push(d);
+        });
+        return () => (el(), push(c));
+    };
 }
 
 @logClassDecoratorRun(0, 19, 21)
 @logClassDecoratorRun(1, 18, 20)
 class A {
-  @logAccessorDecoratorRun(2, 11, 23, 31)
-  @logAccessorDecoratorRun(3, 10, 22, 30)
-  accessor a;
+    @logAccessorDecoratorRun(2, 11, 23, 27)
+    @logAccessorDecoratorRun(3, 10, 22, 26)
+    accessor a;
 
-  @logMethodDecoratorRun(4, 13, 25, 35)
-  @logMethodDecoratorRun(5, 12, 24, 34)
-  b() {};
+    @logMethodDecoratorRun(4, 13, 35, 29)
+    @logMethodDecoratorRun(5, 12, 34, 28)
+    b() {}
 
-  @logMethodDecoratorRun(6, 15, 27, 37)
-  @logMethodDecoratorRun(7, 14, 26, 36)
-  #c() {};
+    @logMethodDecoratorRun(6, 15, 37, 31)
```

Comment:
Is this change correct?
Is this change related to https://github.com/swc-project/swc/issues/7404?

Reply:
irrelevant

This is still incorrect. This PR adjusts the order, but at least it prevents the issue where the decorator initializer does not execute when only fields are present.

Reply:
Then why does this PR modify the execution tests copied from Babel?